addon/layers/-private/base.js

```diff
@@ -79,7 +77,19 @@ export default Ember.Object.extend({
   */
   createSettings() {
     let settings = {
-      opacity: 1
+      opacity: 1,
+      bounds: [
+        [-90, -180],
+        [90, 180]
+      ],
+      wgs84bbox: [
```

Comment:
why not wgs84bbox: null?

Reply:
to correctly bind it in markup, like wgs84bbox.[i].[j]. Look [there](https://github.com/Flexberry/ember-flexberry-gis/blob/feature-144298-layer-extent/addon/templates/components/layers-dialogs/edit.hbs#L167)

Reply:
ok